### ISSUE TITLE
Switch away from Mandrill for sending e-mail before 2016-04-27

### ISSUE THREAD

**Issue by GUI** [18F/api.data.gov#321]
We have pretty light e-mail needs (sending our API key signup confirmation e-mails and contact form messages). We're currently using Mandrill, but they [announced](http://blog.mandrill.com/important-changes-to-mandrill.html) some changes today that mean we need to be tied to a paid MailChimp account to continue using them past March 27. Since I don't think we have any interest in that, and I've been meaning to consolidate all of our stuff to our AWS account for ages now anyway, I think it makes the most sense to switch us over to using Amazon's SES service for sending e-mail.

The extra analytics Mandrill provides are nice and sometimes useful for debugging, but I don't think we really need it, and getting everything consolidated to our single AWS account will be nice. It will also be interesting to see if this might help with some of the e-mail deliverability issues we sometimes would see (although, it seems like those have been pretty quiet as of late): https://github.com/18F/api.data.gov/issues/264

**Comment by pkarman** [comment]
our mailchimp account is paid for already, and we are using mandrill in other apps.

https://18f.slack.com/archives/admins-mailchimp/p1456357560000004

I don't think this needs to change.

**Comment by yepher** [comment]
In case you decide to change SparkPost has a Mandrill migration guide [here](https://www.sparkpost.com/mandrill-migration-guide) and there are very good extra analytics.

**Comment by GUI** [comment]
@pkarman: Thanks for the heads up about the Mandrill account.

I saw some chatter in Slack last week about us bumping into our Mailchimp/Mandrill limits, though, so I went ahead and switched api.data.gov to use AWS SES. Since our e-mail delivery needs are pretty bare bones for api.data.gov, and we don't really use any of the more advanced Mandrill features, this seemed like the easiest option (and I also didn't want to hog Mandrill usage from other 18F projects that might actually need or want the Mandrill features). But let me know if you see any reason for us to use Mandrill instead.

Otherwise, I think this should be taken care of. E-mail is now being sent with SES, and I've checked to ensure we have all the DKIM and SPF stuff setup properly for deliverability.
